app/routes/application.js

```diff
@@ -97,6 +97,23 @@ export default Route.extend(ApplicationRouteMixin, {
           this.set('session.previousRouteName', null);
         }
       });
+      transition.catch(() => {
```

Comment:
this is not the ideal solution, I wrote this on the issue itself, this is like saying if anything at all goes wrong just go to the previous route. This block will be executed when a normal server request returns a 500, you are disconnected from the internet, and a whole lot of other scenarios.